internal/model/record.go

```diff
@@ -7,7 +7,9 @@ import (
 type Record = orderedmapjson.AnyOrderedMap
 
 func NewRecord() *Record {
-	return orderedmapjson.NewAnyOrderedMap()
+	record := orderedmapjson.NewAnyOrderedMap()
+	record.SetEscapeHTML(false)
```

Comment:
can you give an example in what case that the escape html failed?

Reply:
for example I have 2 records from source, represented in JSON below:
```
{"name": "Proctor & Gamble", "established_year": 1987}
{"name": "Unilever", "established_year": 1970}
```
with SetEscapeHTML set to `true`, calls to json.Marshal will return these raw string:
```
{"name": "Proctor \u0026 Gamble", "established_year": 1987}
{"name": "Unilever", "established_year": 1970}
```